projects/demo/src/modules/components/input-month-range/examples/import/insert-template.md

```diff
@@ -0,0 +1,5 @@
+```html
+<form [formGroup]="testForm">
+  <tui-input-month-range formControlName="testValue">Choose a range of months</tui-input-month-range>
```

Comment:
`<input tuiTextfield />` ?

Reply:
It's optional, only if you need to declare something native like `placeholder`. In future we will just have `Textfield` inside `PrimitiveTextfield` as a fallback so it would remain optional in 3.0